src/errors/IISdkCoinErrors.sol

```diff
@@ -1,15 +0,0 @@
-// SPDX-License-Identifier: MIT
```

Comment:
This file should have been removed in https://github.com/cosmos/solidity-ibc-eureka/pull/60, but I missed it.